[Quarkus 2.7] Otplanner tests failing with latest Optaplanner version

I'm trying to bump Optaplanner to latest version merged in the Quarkus platform `8.16.0.Final`. The tests are failing in native mode.

```
[INFO] --- quarkus-maven-plugin:2.7.0.CR1:build (quarkus-build) @ camel-quarkus-integration-test-optaplanner ---
[WARN] *****************************************************************
[WARN] * Your build is requesting parallel execution, but the project  *
[WARN] * relies on System properties at build time which could cause   *
[WARN] * race condition issues thus unpredictable build results.       *
[WARN] * Please avoid using System properties or avoid enabling        *
[WARN] * parallel execution                                            *
[WARN] *****************************************************************
[INFO] [io.quarkus.deployment.pkg.steps.JarResultBuildStep] Building native image source jar: /home/zbendhib/dev/camel-quarkus/integration-tests/optaplanner/target/camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-native-image-source-jar/camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner.jar
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Building native image from /home/zbendhib/dev/camel-quarkus/integration-tests/optaplanner/target/camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-native-image-source-jar/camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner.jar
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Running Quarkus native-image plugin on GraalVM 21.3.0 Java 11 CE (Java Version 11.0.13+7-jvmci-21.3-b05)
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] /home/zbendhib/.sdkman/candidates/java/21.3.0.r11-grl/bin/native-image -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-Dsun.nio.ch.maxUpdateArraySize=100 -J-Dvertx.logger-delegate-factory-class-name=io.quarkus.vertx.core.runtime.VertxLogDelegateFactory -J-Dvertx.disableDnsResolver=true -J-Dio.netty.leakDetection.level=DISABLED -J-Dio.netty.allocator.maxOrder=3 -J-Duser.language=fr -J-Duser.country=FR -J-Dfile.encoding=UTF-8 -H:-ParseOnce -J--add-exports=java.security.jgss/sun.security.krb5=ALL-UNNAMED -J--add-opens=java.base/java.text=ALL-UNNAMED -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy\$BySpaceAndTime -H:+JNI -H:+AllowFoldMethods -J-Djava.awt.headless=true -H:FallbackThreshold=0 -H:+ReportExceptionStackTraces -H:-AddAllCharsets -H:EnableURLProtocols=http -H:NativeLinkerOption=-no-pie -H:-UseServiceLoaderFeature -H:+StackTrace camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner -jar camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner.jar
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]    classlist:   2 666,91 ms,  1,18 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]        (cap):     448,90 ms,  1,18 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]        setup:   1 858,14 ms,  1,18 GB
[WARN] [stderr] java.lang.ClassNotFoundException: org.drools.core.util.Drools$XSTreamChekcer
[WARN] [stderr] 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476)
[WARN] [stderr] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
[WARN] [stderr] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
[WARN] [stderr] 
[WARN] [stderr] 	at com.oracle.svm.hosted.FeatureHandler.forEachFeature(FeatureHandler.java:73)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:688)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:529)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:488)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:403)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:569)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:122)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner$JDK9Plus.main(NativeImageGeneratorRunner.java:599)
[WARN] [stderr] java.lang.ClassNotFoundException: org.drools.compiler.kproject.models.KieModuleMarshaller
[WARN] [stderr] 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:476)
[WARN] [stderr] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
[WARN] [stderr] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
[WARN] [stderr] 	at java.base/java.lang.Class.forName0(Native Method)
[WARN] [stderr] 	at java.base/java.lang.Class.forName(Class.java:398)
[WARN] [stderr] 	at io.quarkus.runner.AutoFeature.beforeAnalysis(Unknown Source)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.lambda$runPointsToAnalysis$11(NativeImageGenerator.java:688)
[WARN] [stderr] 	at com.oracle.svm.hosted.FeatureHandler.forEachFeature(FeatureHandler.java:73)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:688)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:529)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:488)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:403)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:569)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:122)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner$JDK9Plus.main(NativeImageGeneratorRunner.java:599)
[WARN] [stderr] java.lang.NullPointerException
[WARN] [stderr] 	at org.graalvm.sdk/org.graalvm.nativeimage.hosted.RuntimeClassInitialization.getUnqualifiedName(RuntimeClassInitialization.java:176)
[WARN] [stderr] 	at org.graalvm.sdk/org.graalvm.nativeimage.hosted.RuntimeClassInitialization.initializeAtRunTime(RuntimeClassInitialization.java:99)
[WARN] [stderr] 	at io.quarkus.runner.AutoFeature.beforeAnalysis(Unknown Source)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.lambda$runPointsToAnalysis$11(NativeImageGenerator.java:688)
[WARN] [stderr] 	at com.oracle.svm.hosted.FeatureHandler.forEachFeature(FeatureHandler.java:73)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:688)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:529)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:488)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:403)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:569)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:122)
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner$JDK9Plus.main(NativeImageGeneratorRunner.java:599)
[WARN] [stderr] 	at java.base/java.lang.Class.forName0(Native Method)
[WARN] [stderr] 	at java.base/java.lang.Class.forName(Class.java:398)
[WARN] [stderr] 	at io.quarkus.runner.AutoFeature.beforeAnalysis(Unknown Source)
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]     (clinit):     595,12 ms,  5,62 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]   (typeflow):   1 974,35 ms,  5,62 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]    (objects):  15 320,53 ms,  5,62 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]   (features):   3 433,08 ms,  5,62 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]     analysis:  22 684,54 ms,  5,62 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]     universe:   1 640,21 ms,  5,62 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]      (parse):   4 139,85 ms,  5,63 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]     (inline):   6 527,55 ms,  6,33 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]    (compile):  20 310,29 ms,  6,44 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]      compile:  34 240,88 ms,  6,44 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]        image:   3 275,37 ms,  6,44 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]        write:     567,63 ms,  6,44 GB
[INFO] [stdout] [camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner:102200]      [total]:  67 320,80 ms,  6,44 GB
[INFO] [stdout] # Printing build artifacts to: /home/zbendhib/dev/camel-quarkus/integration-tests/optaplanner/target/camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-native-image-source-jar/camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner.build_artifacts.txt
[WARN] [stderr] 	at com.oracle.svm.hosted.NativeImageGenerator.lambda$runPointsToAnalysis$11(NativeImageGenerator.java:688)
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] objcopy --strip-debug camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner
[INFO] [io.quarkus.deployment.QuarkusAugmentor] Quarkus augmentation completed in 69859ms
[INFO] 
[INFO] --- maven-failsafe-plugin:2.22.2:integration-test (default) @ camel-quarkus-integration-test-optaplanner ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.apache.camel.quarkus.component.optaplanner.it.OptaplannerIT
[WARN] [stderr] janv. 24, 2022 5:02:18 PM org.jboss.threads.Version <clinit>
[WARN] [stderr] INFO: JBoss Threads version 3.4.2.Final
[WARN] [stderr] ======================================================================================
[WARN] [stderr]   native.image.path was not set, making a guess for the correct path of native image
[WARN] [stderr]   guessed path: /home/zbendhib/dev/camel-quarkus/integration-tests/optaplanner/target/camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner
[WARN] [stderr] ======================================================================================
[INFO] [stdout] Executing "/home/zbendhib/dev/camel-quarkus/integration-tests/optaplanner/target/camel-quarkus-integration-test-optaplanner-2.7.0-SNAPSHOT-runner -Dquarkus.http.port=44809 -Dquarkus.http.ssl-port=42791 -Dtest.url=http://localhost:44809 -Dquarkus.log.file.path=/home/zbendhib/dev/camel-quarkus/integration-tests/optaplanner/target/quarkus.log -Dquarkus.log.file.enable=true"
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 1, Time elapsed: 1.425 s <<< FAILURE! - in org.apache.camel.quarkus.component.optaplanner.it.OptaplannerIT
[ERROR] solveSync  Time elapsed: 0.008 s  <<< ERROR!
java.lang.RuntimeException: Unable to successfully launch process '102632'. Exit code is: '1'.
	at io.quarkus.test.common.LauncherUtil.ensureProcessIsAlive(LauncherUtil.java:98)
	at io.quarkus.test.common.LauncherUtil.waitForCapturedListeningData(LauncherUtil.java:67)
	at io.quarkus.test.common.DefaultNativeImageLauncher.start(DefaultNativeImageLauncher.java:107)
	at io.quarkus.test.junit.IntegrationTestUtil.startLauncher(IntegrationTestUtil.java:183)
	at io.quarkus.test.junit.NativeTestExtension.doNativeStart(NativeTestExtension.java:163)
	at io.quarkus.test.junit.NativeTestExtension.ensureStarted(NativeTestExtension.java:102)
	at io.quarkus.test.junit.NativeTestExtension.beforeAll(NativeTestExtension.java:75)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$10(ClassBasedTestDescriptor.java:381)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:381)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:205)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:80)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:148)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   OptaplannerIT.solveSync » Runtime Unable to successfully launch process '10263...
[INFO] 
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 1
[INFO] 
[INFO] 
[INFO] --- maven-failsafe-plugin:2.22.2:verify (default) @ camel-quarkus-integration-test-optaplanner ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:27 min (Wall Clock)
[INFO] Finished at: 2022-01-24T17:02:19+01:00
[INFO] ------------------------------------------------------------------------

```

--- a/extensions/optaplanner/deployment/src/main/java/org/apache/camel/quarkus/component/optaplanner/deployment/OptaplannerProcessor.java
+++ b/extensions/optaplanner/deployment/src/main/java/org/apache/camel/quarkus/component/optaplanner/deployment/OptaplannerProcessor.java
@@ -16,10 +16,8 @@
  */
 package org.apache.camel.quarkus.component.optaplanner.deployment;
 
-import io.quarkus.deployment.annotations.BuildProducer;
 import io.quarkus.deployment.annotations.BuildStep;
 import io.quarkus.deployment.builditem.FeatureBuildItem;
-import io.quarkus.deployment.builditem.nativeimage.RuntimeInitializedClassBuildItem;
 
 class OptaplannerProcessor {
 
@@ -30,11 +28,4 @@ class OptaplannerProcessor {
         return new FeatureBuildItem(FEATURE);
     }
 
-    @BuildStep
-    void runtimeInitializedClasses(BuildProducer<RuntimeInitializedClassBuildItem> runtimeInitializedClass) {
-        runtimeInitializedClass.produce(new RuntimeInitializedClassBuildItem("org.drools.core.util.Drools$XSTreamChekcer"));
-        runtimeInitializedClass
-                .produce(new RuntimeInitializedClassBuildItem("org.drools.compiler.kproject.models.KieModuleMarshaller"));
-        runtimeInitializedClass.produce(new RuntimeInitializedClassBuildItem("org.drools.core.rule.JavaDialectRuntimeData"));
-    }
 }

--- a/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/bootstrap/DataGenerator.java
+++ b/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/bootstrap/DataGenerator.java
@@ -32,8 +32,8 @@ import org.apache.camel.quarkus.component.optaplanner.it.domain.Timeslot;
 
 /**
  * adapted from optaplanner quarkus quickstart :
- * https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/quarkus-school-timetabling/src/main/java/org/acme/schooltimetabling/bootstrap/DemoDataGenerator.java
- */
+ * https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
+ **/
 @ApplicationScoped
 public class DataGenerator {
 

--- a/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/domain/Lesson.java
+++ b/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/domain/Lesson.java
@@ -24,7 +24,7 @@ import org.optaplanner.core.api.domain.variable.PlanningVariable;
 
 /**
  * adapted from optaplanner quarkus quickstart :
- * https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/quarkus-school-timetabling/src/main/java/org/acme/schooltimetabling/domain/Lesson.java
+ * https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
  */
 @PlanningEntity
 public class Lesson {

--- a/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/domain/Room.java
+++ b/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/domain/Room.java
@@ -22,7 +22,7 @@ import org.optaplanner.core.api.domain.lookup.PlanningId;
 
 /**
  * adapted from optaplanner quarkus quickstart :
- * https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/quarkus-school-timetabling/src/main/java/org/acme/schooltimetabling/domain/Room.java
+ * https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
  */
 public class Room {
 

--- a/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/domain/TimeTable.java
+++ b/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/domain/TimeTable.java
@@ -27,7 +27,7 @@ import org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScore;
 
 /**
  * adapted from optaplanner quarkus quickstart :
- * https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/quarkus-school-timetabling/src/main/java/org/acme/schooltimetabling/domain/TimeTable.java
+ * https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
  */
 @PlanningSolution
 public class TimeTable {

--- a/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/domain/Timeslot.java
+++ b/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/domain/Timeslot.java
@@ -24,7 +24,7 @@ import org.optaplanner.core.api.domain.lookup.PlanningId;
 
 /**
  * adapted from optaplanner quarkus quickstart :
- * https://github.com/kiegroup/optaplanner-quickstarts/blob/stable/quarkus-school-timetabling/src/main/java/org/acme/schooltimetabling/domain/Timeslot.java
+ * https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
  */
 public class Timeslot {
 

--- a/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/solver/TimeTableConstraintProvider.java
+++ b/integration-tests/optaplanner/src/main/java/org/apache/camel/quarkus/component/optaplanner/it/solver/TimeTableConstraintProvider.java
@@ -27,7 +27,7 @@ import org.optaplanner.core.api.score.stream.Joiners;
 
 /**
  * adapted from optaplanner quarkus quickstart :
- * https://github.com/quarkusio/quarkus-quickstarts/blob/master/optaplanner-quickstart/src/main/java/org/acme/optaplanner/solver/TimeTableConstraintProvider.java
+ * https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
  */
 public class TimeTableConstraintProvider implements ConstraintProvider {
 

--- a/integration-tests/optaplanner/src/main/resources/application.properties
+++ b/integration-tests/optaplanner/src/main/resources/application.properties
@@ -17,7 +17,7 @@
 
 ########################
 # OptaPlanner properties
-# adapted from optaplanner quarkus quickstart : https://github.com/quarkusio/quarkus-quickstarts/blob/master/optaplanner-quickstart/src/main/resources/application.properties
+# adapted from optaplanner quarkus quickstart : https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
 ########################
 
 # The solver runs for 30 seconds. To run for 5 minutes use "5m" and for 2 hours use "2h".

--- a/integration-tests/optaplanner/src/test/java/org/apache/camel/quarkus/component/optaplanner/it/OptaplannerIT.java
+++ b/integration-tests/optaplanner/src/test/java/org/apache/camel/quarkus/component/optaplanner/it/OptaplannerIT.java
@@ -17,10 +17,7 @@
 package org.apache.camel.quarkus.component.optaplanner.it;
 
 import io.quarkus.test.junit.NativeImageTest;
-import org.junit.jupiter.api.condition.DisabledIfEnvironmentVariable;
 
-// https://github.com/apache/camel-quarkus/issues/2205
-@DisabledIfEnvironmentVariable(named = "CI", matches = "true")
 @NativeImageTest
 class OptaplannerIT extends OptaplannerTest {
 

--- a/integration-tests/optaplanner/src/test/java/org/apache/camel/quarkus/component/optaplanner/it/OptaplannerTest.java
+++ b/integration-tests/optaplanner/src/test/java/org/apache/camel/quarkus/component/optaplanner/it/OptaplannerTest.java
@@ -22,13 +22,10 @@ import io.quarkus.test.junit.QuarkusTest;
 import io.restassured.RestAssured;
 import org.apache.camel.quarkus.component.optaplanner.it.domain.TimeTable;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.condition.DisabledIfEnvironmentVariable;
 
 import static org.awaitility.Awaitility.await;
 import static org.hamcrest.Matchers.notNullValue;
 
-// https://github.com/apache/camel-quarkus/issues/2205
-@DisabledIfEnvironmentVariable(named = "CI", matches = "true")
 @QuarkusTest
 class OptaplannerTest {
 

--- a/pom.xml
+++ b/pom.xml
@@ -47,7 +47,7 @@
         <camel.sb.docs.branch>camel-spring-boot-${camel.major.minor}.x</camel.sb.docs.branch><!-- The stable camel-spring-boot branch on which our Antora docs depends -->
         <cassandra-quarkus.version>1.1.1</cassandra-quarkus.version><!-- https://repo1.maven.org/maven2/com/datastax/oss/quarkus/cassandra-quarkus-bom/ -->
         <debezium.version>1.6.1.Final</debezium.version><!-- May go back to Camel's ${debezium-version} when they are in sync https://repo1.maven.org/maven2/io/debezium/debezium-bom/ -->
-        <optaplanner.version>8.14.0.Final</optaplanner.version><!-- May go back to Camel's ${optaplanner-version} when they are in sync https://repo1.maven.org/maven2/org/optaplanner/optaplanner-quarkus/ -->
+        <optaplanner.version>8.16.0.Final</optaplanner.version><!-- May go back to Camel's ${optaplanner-version} when they are in sync https://repo1.maven.org/maven2/org/optaplanner/optaplanner-quarkus/ -->
         <quarkiverse-amazonservices.version>1.0.3</quarkiverse-amazonservices.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/amazonservices/quarkus-amazon-services-parent/ -->
         <quarkiverse-artemis.version>1.0.3</quarkiverse-artemis.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/artemis/quarkus-artemis-parent/ -->
         <quarkiverse-freemarker.version>0.3.0</quarkiverse-freemarker.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/freemarker/quarkus-freemarker-parent/ -->

